src/middlewared/middlewared/plugins/disk.py

```diff
@@ -1205,6 +1205,31 @@ def gptid_from_part_type(self, disk, part_type):
             raise ValueError(f'Partition type {part_type} not found on {disk}')
         return f'gptid/{uuid.text}'
 
+    @private
+    def unlabel(self, disk):
+        self.middleware.call_sync('disk.swaps_remove_disks', [disk])
+
+        subprocess.run(
```

Comment:
`check=True` everywhere or isn't it necessary?

Reply:
@themylogin I dont believe this is necessary, this is just moving the old code which doesnt check this. I believe there are things that would prevent it from working 100% reliably and fail here can be ignored.